src/layout/default-layout.vue

```diff
@@ -13,12 +13,31 @@ a-layout.layout-container
   import { useAppStore } from '@/store'
   import Footer from '@/components/footer/index.vue'
   import useResponsive from '@/hooks/responsive'
+  import dayjs from 'dayjs'
+  import utc from 'dayjs/plugin/utc'
+  import timezone from 'dayjs/plugin/timezone'
   import PageLayout from './page-layout.vue'
 
+  dayjs.extend(utc)
+  dayjs.extend(timezone)
+
   useResponsive(true)
   const navbarHeight = `52px`
 
-  const { navbar, footer } = storeToRefs(useAppStore())
+  const { navbar, footer, userTimezone } = storeToRefs(useAppStore())
+
+  onMounted(() => {
+    // TODO: Use default timezone or timezone from local storage?
+    // Get default timezone
+    const offsetMinutes = dayjs.tz(dayjs()).utcOffset()
```

Comment:
Consider Daylight time, it's not a good way to get the timezone in this way.

https://stackoverflow.com/questions/6939685/get-client-time-zone-from-browser

Reply:
dayjs is using the same thing. https://day.js.org/docs/en/timezone/timezone
The result is already DST offsets from UTC.